internal/pkg/deploy/cloudformation/stack/transformers_test.go

```diff
@@ -2091,7 +2091,22 @@ func Test_convertDeploymentConfig(t *testing.T) {
 			out: template.DeploymentConfigurationOpts{
 				MinHealthyPercent: minHealthyPercentDefault,
 				MaxPercent:        maxPercentDefault,
-				RollbackAlarms:    []string{"alarmName1", "alarmName2"},
+				RollbackAlarms:    "[alarmName1, alarmName2]",
```

Comment:
Why did we change `RollbackAlarms` to a string from the slice?

Reply:
Good question-- for ease of use in the CFN template. `FmtSliceFunc` adds the brackets and commas.

Reply:
I could also switch it back and use `range` in the CFN template.

Reply:
We can probably keep it as `[]string` and call `fmtSliceFunc .Alarms` like this https://github.com/aws/copilot-cli/blob/3f458da01ce1cb412530c84508af9b4eda1e1c2a/internal/pkg/template/templates/workloads/partials/cf/https-listener.yml#L109